store/tikv/kv/store_vars.go

```diff
@@ -25,15 +25,14 @@ type ReplicaReadType byte
 
 const (
 	// ReplicaReadLeader stands for 'read from leader'.
-	ReplicaReadLeader ReplicaReadType = 1 << iota
+	ReplicaReadLeader ReplicaReadType = iota
```

Comment:
We treat both `0` and `1` previously as `ReplicaReadLeader` so for `ReplicaReadLeader` it is ok. But now `ReplicaReadFollower` becomes `1` which might cause problem.

Did you check the usage of `ReplicaReadType`? Are they all from `IsFollowerRead`? If so, then it is ok to do this change.

Reply:
It already causes CI to fail, please fix it.

Reply:
fixed. It causes problem if we use `0x01` instead of `ReplicaReadLeader`.